coalib/bears/Bear.py

```diff
@@ -24,6 +24,26 @@
 from .meta import bearclass
 
 
+def _debugger_setting_value(bear):
+    """
+    Determines the value of ``debugger`` setting.
+
+    :param bear: Bear object.
+    :return:     True if ``debug_bears`` is ``True`` or if bear name specified
+                 in ``debug_bears`` setting match with the bear parameter.
+    """
+    if bear is None:
+        return False
```

Comment:
Is that actually even valid passing `None` for a bear? We are determining if nothing is being debugged or not? :P